package.json

```diff
@@ -1,14 +1,14 @@
 {
-  "name": "mdn-browser-compat-data",
-  "version": "1.0.37",
+  "name": "@mdn/browser-compat-data",
+  "version": "2.0.0-1",
```

Comment:
Is the dash correct here?

Reply:
This marks the release as a semver pre-release, so yes. I could've used something like `2.0.0-alpha1` but I was lazy and used the default generated by `npm version prerelease`. If I do any more prereleases, I'll do that, but this ought to be a one-time thing.

Reply:
OK, cool.